app/views/users/index.html.erb

```diff
@@ -5,6 +5,7 @@
         Name: <%= user.name %>
         <span class="profile-link">
           <%= link_to 'See Profile',  user_path(user), class: 'profile-link' %>
+          <%= button_to "Invite to Friendship",friendships_path(user_id:user)%>
```

Comment:
This button shouldn't appear when you are looking at your own profile. I see that you have a "solution" to manage that, but you need to improve it by removing it. This is better for the UX.

The button's text is too long and you can't see it properly. Please make the button wider.

The button should change when you already sent an invitation to someone or when I'm already their friend.. This can be achieved by creating two different buttons and render them conditionally, depending on the friendship status.